packages/aws-client/src/S3.ts

```diff
@@ -1003,6 +1003,13 @@ const uploadPartCopy = async (
  * @param {number} [params.chunkSize] - chunk size of the S3 multipart uploads
  * @returns {Promise.<{ etag: string }>} object containing the ETag of the
  *    destination object
+ *
```

Comment:
Do we need so many comments here?  It should be enough to just add 'the method is for non-zero size file only'.
The error message  mentioned could change or no longer valid in the future, and can cause confusion.

Reply:
I think that the MalformedXML error is a misleading error message from AWS, and that's why I thought it might be helpful to have the information. I also generally prefer a bit more information than less. But I agree the error messages may change, and as long as someone can find the references they can find the history and the error messages encountered, so the comments can be trimmed down.